src/test/isolation2/expected/resgroup/resgroup_alter_concurrency.out

```diff
@@ -368,7 +368,7 @@ DROP RESOURCE GROUP rg_concurrency_test;
 DROP
 
 61<:  <... completed>
-ERROR:  role with Oid 213301 was dropped
+ERROR:  role with Oid -2147450804 was dropped
```

Comment:
-2147450804?

Reply:
i guess it cause by src/backend/commands/resgroupcmds.c:L695

```
ereport(ERROR,
				(errcode(ERRCODE_UNDEFINED_OBJECT),
				errmsg("role with Oid %d was dropped", roleid),
				errdetail("Cannot execute commands anymore, please terminate this session.")));
```

should be `%u` here, let me double check and fix it....